### ISSUE TITLE
Watcher not recompiling on changes on Windows

### ISSUE THREAD

**Issue by Fjormos** [kentaroi/eleventy-sass#12]
Hey, great plugin!
I just discovered a bug to similar #3, where eleventy-sass does not recompile .scss files on changes in the file on Windows.

If setting the DEBUG env env variable, i get the following output:
```
[11ty] File changed: src\stylesheets\main.scss
  Dev:EleventySass eleventy.beforeWatch queue: [ './src\\stylesheets\\main.scss' ] +6s
  Dev:EleventySass getCacheKey(contents, ./src/stylesheets/main.scss) -> src/stylesheets/main.scss2022-08-07T16:59:06.736Zinitial +30ms
```

My dependencies:
- @11ty/eleventy@1.0.1
- eleventy-sass@2.1.0

I used node in version 16.16.0

I also set up a minimal project [here](https://github.com/Fjormos/eleventy-sass-bug)

**Comment by kentaroi** [comment]
Thank you very much for filing the issue and providing the minimal project.
I'm sorry for the late reply.
I reproduced it on Windows.

I'll fix it in a day or two.

**Comment by kentaroi** [comment]
I just fixed it and published v2.1.1.
